docs/README.md

```diff
@@ -1,217 +1,86 @@
-# Kapitan: Generic templated configuration management for Kubernetes, Terraform and other things
+# :kapitan-logo: Kapitan: advanced configuration management tool
 
-![Unit Tests](https://github.com/kapicorp/kapitan/actions/workflows/test.yml/badge.svg)
```

Comment:
We should keep these badges should still be at the top

Reply:
ok for the github page, but even for the website? Looks ugly. That's more a github thing?

Reply:
I have left it as it is in the github repo README, but on the website I have moved it to more relevant places. I don't think it belongs to the website

Reply:
got it